src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/entity/Assessment.kt

```diff
@@ -80,6 +81,10 @@ data class Assessment(
   @Column(name = "address_checks_complete")
   var addressChecksComplete: Boolean = false,
 
+  @NotNull
+  @Enumerated(EnumType.STRING)
+  var addressChecksStatus: ResidentialChecksStatus? = null,
```

Comment:
I don't think this needs to be null right!

`var addressChecksStatus: ResidentialChecksStatus = NOT_STARTED,`